en/flight_controller/modalai_fc_v1.md

```diff
@@ -50,23 +35,38 @@ This board supported in QGroundControl 4.0 and later.
 |                  | I2C |
 |                  | GPIO |
 
+> **Note** More detailed hardware documentation can be found [here](https://docs.modalai.com/flight-core-datasheet/).
+
 ## Dimensions
 
 ![FlightCoreV1Dimensions](../../assets/flight_controller/modalai/fc_v1/dimensions.png)
 
+
+## PX4 Firwmare Compatibility
+
+Although supported in PX4 v1.10, a few issues have been addressed since then that are currently being maintained by ModalAI [here](https://github.com/modalai/px4-firmware/tree/modalai-1.10).
```

Comment:
@modaltb FYI I have just flattened the document structure and ordering of docs in order to have consistent approach for all manufacturers.

As part of that I saw this section on compatibility. 
We're about to update to PX4 v1.11. Is there any new "story" here? I will be taking a cut of the PX4 v1.11 docs shortly, so if there are changes would be good to capture them sooner rather than later.

Reply:
Hey I just created a small PR that hopefully captures what's needed for v1.11, thanks!